ecs/indexer.container-definition.json

```diff
@@ -0,0 +1,143 @@
+[
+  {
+    "name": "{{Name}}",
+    "image": "{{Image}}",
+    "memory": 1536,
+    "portMappings": [
+      {
+        "containerPort": 3000,
+        "hostPort": 0,
+        "protocol": "tcp"
+      }
+    ],
+    "essential": true,
+    "entryPoint": [
+      "yarn",
+      "docker:start:indexer"
+    ],
+    "environment": [
+      {
+        "name": "NEAR_WALLET_ENV",
+        "value": "{{NearWalletEnvironment}}"
+      },
+      {
+        "name": "ENABLE_PHONE_IDENTITY_VERIFICATION",
+        "value": "true"
+      },
+      {
+        "name": "AWS_REGION",
+        "value": "{{AwsRegion}}"
+      },
+      {
+        "name": "PORT",
+        "value": "3000"
+      },
+      {
+        "name": "NEW_ACCOUNT_AMOUNT",
+        "value": "10000000000000000000000000"
+      },
+      {
+        "name": "FUNDED_ACCOUNT_BALANCE",
+        "value": "350000000000000000000000"
+      },
+      {
+        "name": "DEBUG",
+        "value": "koa-ratelimit"
+      },
+      {
+        "name": "TOKEN_CONTRACTS",
+        "value": "meta.pool.testnet"
+      },
+      {
+        "name": "GAS_2FA_CONFIRM",
+        "value": "250000000000000"
+      },
+      {
+        "name": "FUNDED_NEW_ACCOUNT_CONTRACT_NAME",
+        "value": "near"
+      },
+      {
+        "name": "NODE_URL",
+        "value": "https://rpc.testnet.near.org"
+      },
+      {
+        "name": "DISABLE_PHONE_RECOVERY",
+        "value": "true"
+      },
+      {
+        "name": "ENABLE_EMAIL_IDENTITY_VERIFICATION",
+        "value": "true"
+      }
+    ],
+    "secrets": [
+      {
+        "name": "ACCOUNT_CREATOR_KEY",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/ACCOUNT_CREATOR_KEY"
+      },
+      {
+        "name": "ACCOUNT_CREATOR_KEYS",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/ACCOUNT_CREATOR_KEYS"
+      },
+      {
+        "name": "BLOCK_TEMP_EMAIL_API_KEY",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/BLOCK_TEMP_EMAIL_API_KEY"
+      },
+      {
+        "name": "FUNDED_ACCOUNT_CREATOR_KEY",
+        "valueFrom": "arn:aws:ssm:{{AwsRegion}}:{{AwsAccountId}}:parameter/env/{{Environment}}/FUNDED_ACCOUNT_CREATOR_KEY"
+      },
+      {
+        "name": "INDEXER_DB_CONNECTION",
```

Comment:
I think this might be the only variable we need for `indexer`

Reply:
Just this and a couple more. I've removed all which are unused within `indexer` :)